services/src/main/java/org/fao/geonet/api/site/SiteApi.java

```diff
@@ -206,37 +206,28 @@ public SettingsListResponse get(
             Settings.SYSTEM_PLATFORM_VERSION,
             Settings.SYSTEM_PLATFORM_SUBVERSION
         }));
-        if (!NodeInfo.DEFAULT_NODE.equals(node.getId())) {
-            Source source = sourceRepository.findOne(node.getId());
-            if (source != null) {
-                String iso3langCode = languageUtils.getIso3langCode(request.getLocales());
-                final List<Setting> settings = response.getSettings();
-                settings.add(
-                    new Setting().setName(Settings.NODE_DEFAULT)
-                        .setValue("false"));
-                settings.add(
-                    new Setting().setName(Settings.NODE)
-                        .setValue(source.getUuid()));
-                settings.add(
-                    new Setting().setName(Settings.NODE_NAME)
-                        .setValue(source != null ? source.getLabel(iso3langCode) : source.getName()));
-            }
+        Source source;
+        String nodeDefault;
+        if (NodeInfo.DEFAULT_NODE.equals(node.getId())) {
+            source = sourceRepository.findOne(settingManager.getSiteId());
+            nodeDefault = "true";
         } else {
-            String defaultSiteId = settingManager.getSiteId();
-            Source source = sourceRepository.findOne(defaultSiteId);
-            if (source != null) {
-                String iso3langCode = languageUtils.getIso3langCode(request.getLocales());
-                final List<Setting> settings = response.getSettings();
-                settings.add(
-                    new Setting().setName(Settings.NODE_DEFAULT)
-                        .setValue("true"));
-                settings.add(
-                    new Setting().setName(Settings.NODE)
-                        .setValue(NodeInfo.DEFAULT_NODE));
-                settings.add(
-                    new Setting().setName(Settings.NODE_NAME)
-                        .setValue(source != null ? source.getLabel(iso3langCode) : source.getName()));
-            }
+            source = sourceRepository.findOne(node.getId());
+            nodeDefault = "false";
+        }
+        if (source != null) {
+            final List<Setting> settings = response.getSettings();
+            String iso3langCode = languageUtils.getIso3langCode(request.getLocales());
+
+            settings.add(
+                new Setting().setName(Settings.NODE_DEFAULT)
+                    .setValue(nodeDefault));
+            settings.add(
+                new Setting().setName(Settings.NODE)
+                    .setValue(source.getUuid()));
```

Comment:
In previous version for the default node this value was `srv`, but with this change it's the uuid for the related source.

Previous response:

`..."node/id":"srv","node/name":"My GeoNetwork catalogue"}`

With the change:

`..."node/id":"eb8fd951-22df-4686-8c47-fca1361bf9f6","node/name":"My GeoNetwork catalogue"}`

Reply:
That is what this change does.

In a new fresh install, the site image is broken because `srv.png` does not exists - but `eb8fd951-22df-4686-8c47-fca1361bf9f6.png` does exists.  

I'm not sure why the default `node/id` was `srv` while the others are the uuid?

Are you saying that having the `node/id` as a uuid is causing problems?  If so then the problem may be with the creation of the default `eb8fd951-22df-4686-8c47-fca1361bf9f6.png` image which should be `svn.png`?